lib/maid/app.rb

```diff
@@ -155,6 +164,59 @@ def migrate_trash
     end
   end
 
+  def install_launch_agent
+    maid_executable = %x(which maid)
+    plist_destination = File.expand_path("~/Library/LaunchAgents/benjaminoakes.maid.plist")
```

Comment:
I own [benjaminoakes.com](http://benjaminoakes.com).  Should this be `com.benjaminoakes.maid` instead?